flink/v1.15/build.gradle

```diff
@@ -134,7 +133,9 @@ project(":iceberg-flink:iceberg-flink-runtime-${flinkMajorVersion}") {
   }
 
   dependencies {
-    implementation project(":iceberg-flink:iceberg-flink-${flinkMajorVersion}")
+    implementation(project(":iceberg-flink:iceberg-flink-${flinkMajorVersion}")) {
+      exclude group: 'org.apache.flink'
```

Comment:
I assume this moved from 124 as this is more desired/precise style, right?

Reply:
yes, because otherwise if the exlude is configured for the entire `implementation` configuration, the `org.apache.flink:flink-metrics-dropwizard` will never make it into the runtime jar (because it's excluded).